packages/rest/src/rest.server.ts

```diff
@@ -650,11 +681,13 @@ export class RestServer extends Context implements Server, HttpServerLike {
     );
   }
 
-  private bindRoute(r: RouteEntry, namespace = 'routes') {
+  private bindRoute(r: RouteEntry) {
+    const namespace = RestBindings.ROUTES;
```

Comment:
Does the change here mean we don't allow custom namespace anymore?

Reply:
Oh, the PR that introduces `bindRoute` has not been released yet. So there is no breaking change.